cloudcix_primitives/snapshot_lxd.py

```diff
@@ -14,18 +15,354 @@
+
+        instance = ret['payload_message']
+        fmt.add_successful('containers.get', ret)
+        
+        # Find and delete the snapshot
+        try:
+            # Get the snapshot
+            snapshot = instance.snapshots.get(snapshot_name)
+            if not snapshot:
+                fmt.add_successful('snapshot.not_found', {'snapshot_name': snapshot_name})
+                return True, f"1100: {messages[1100]}", fmt.successful_payloads
+            
+            # Delete the snapshot with wait=True to ensure the operation completes before continuing
+            snapshot.delete(wait=True)
+            fmt.add_successful('snapshot.delete', {'snapshot_name': snapshot_name})
+            
+        except Exception as e:
+            return False, f"{prefix+4}: {messages[3124]}{e}", fmt.successful_payloads
+            
+        return True, '', fmt.successful_payloads
+
+    status, msg, successful_payloads = run_host(endpoint_url, 3120, {})
+
+    if status is False:
+        return status, msg, successful_payloads
+
+    return True, f'1100: {messages[1100]}'
 
-def build() -> Tuple[bool, str]:
-    return(False, 'Not Implemented')
 
+def update(
+        endpoint_url: str,
+        project: str,
+        container_name: str,
+        snapshot_name: str,
+        verify_lxd_certs: bool = True
+) -> Tuple[bool, str]:
+    """
+    description:
+        Restore a container from a snapshot.
+        NOTE: Since we're using ZFS filesystem, only one snapshot can exist at a time,
+        so the snapshot_name parameter is required to identify the snapshot.
+    
+    parameters:
+        endpoint_url:
+            description: The endpoint URL for the LXD Host.
+            type: string
+            required: true
+        project: 
+            description: Unique identification name of the LXD Project on the LXD Host.
+            type: string
+            required: true
+        container_name:
+            description: The name of the LXD container containing the snapshot.
+            type: string
+            required: true
+        snapshot_name:
+            description: The name of the snapshot to restore from.
+            type: string
+            required: true
+        verify_lxd_certs:
+            description: Boolean to verify LXD certs.
+            type: boolean
+            required: false
+    
+    return:
+        description: |
+            A tuple with a boolean flag indicating success or failure, a message, and a dictionary of successful payloads.
+        type: tuple
+    """
+    # Define messages
+    messages = {
+        1300: f'Successfully restored container {container_name} from snapshot {snapshot_name} on {endpoint_url}',
+        3321: f'Failed to connect to {endpoint_url} for containers.get payload',
+        3322: f'Failed to run containers.get payload on {endpoint_url}',
+        3323: f'No snapshots found for container {container_name}',
```

Comment:
```suggestion
```

Not used so removing, the below should be renumbered. I suggest moving `3325` to `3323` as it is the third error that potentially could be encountered